README.md

```diff
@@ -1,4 +1,56 @@
-# ceph-bootstrap - CLI tool to deploy Ceph clusters <br/> [![Build Status](https://travis-ci.com/SUSE/ceph-bootstrap.svg?branch=master)](https://travis-ci.com/SUSE/ceph-bootstrap) [![codecov](https://codecov.io/gh/SUSE/ceph-bootstrap/branch/master/graph/badge.svg)](https://codecov.io/gh/SUSE/ceph-bootstrap)
+# ceph-bootstrap - Deploy Ceph clusters using cephadm<br/> [![Build Status](https://travis-ci.com/SUSE/ceph-bootstrap.svg?branch=master)](https://travis-ci.com/SUSE/ceph-bootstrap) [![codecov](https://codecov.io/gh/SUSE/ceph-bootstrap/branch/master/graph/badge.svg)](https://codecov.io/gh/SUSE/ceph-bootstrap)
 
-`ceph-bootstrap` is a CLI tool to deploy Ceph clusters (both the upstream and SUSE
-downstream versions) using Salt.
+This project is aimed at deploy [Ceph][ceph] clusters managed by
```

Comment:
I prefer the old beginning of the sentence: 

`ceph-bootstrap` is a CLI tool for deploying Ceph clusters . . .

Reply:
well, the repository also contains ceph-salt-formula that is not a cli

Reply:
Ah, I see. So:

This project provides tools for deploying Ceph clusters . . .

Reply:
Actually, what is this repository?

* It's not a set of independent tools
* It's not just a CLI tool
* It's not just a Salt Formula

I'd say it's a "project"

Reply:
I agree - repeating my new suggestion from above:

```
This project provides tools for deploying Ceph clusters . . .
```

which I believe sounds better in English than "This project is aims to deploy Ceph clusters"

Reply:
Another possibility could be:

```
The aim of this project is to deploy Ceph clusters
```

but that sounds weird to my ears.

Reply:
rephrased it to: 

> This project provids a complete workflow to deploy [Ceph][ceph] clusters managed by [cephadm][cephadm] using [Salt][salt]

Reply:
I have some trouble to understand what "complete workflow" means in that sentence. I prefer the more simple suggestion that @smithfarm gave: "This project provides tools for deploying Ceph clusters . . ."

Reply:
fixed